<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="SoftwareManagement.sm_apt_install_download_path" skipped="0" tests="1" time="83.42" timestamp="2022-05-20T12:12:28">
	<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED"/>
		<system-out>12:11:04 INFO  ==============================================================
12:11:04 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
12:11:04 INFO  Dir:   software_management_end_to_end/
12:11:04 INFO  Title: Install package to directory specified in tedge config set tmp.path
12:11:04 INFO  ==============================================================
12:11:06 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
12:11:06 INFO  EnvironmentC8y Setup
12:11:06 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
12:11:20 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:11:21 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
12:11:22 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
12:11:22 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
12:11:24 INFO  State of current operation: FAILED
12:11:25 INFO  Assert that (False == value) with value=False ... passed
12:11:25 INFO  Executed sudo, exit status 0
12:11:26 INFO  Executed sudo, exit status 0
12:11:26 INFO  Executed sudo, exit status 0
12:11:30 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:11:31 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
12:11:31 INFO  Delaying execution by 7 seconds
12:11:52 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
12:11:52 INFO  Response status: 201
12:11:52 INFO  Response to action: {
    &quot;creationTime&quot;: &quot;2022-05-20T12:11:52.229Z&quot;,
    &quot;deviceId&quot;: &quot;19392643&quot;,
    &quot;deviceName&quot;: &quot;m32sd10a&quot;,
    &quot;self&quot;: &quot;https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409399&quot;,
    &quot;id&quot;: &quot;19409399&quot;,
    &quot;status&quot;: &quot;PENDING&quot;,
    &quot;description&quot;: &quot;Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]&quot;,
    &quot;c8y_SoftwareUpdate&quot;: [
        {
            &quot;name&quot;: &quot;rolldice&quot;,
            &quot;action&quot;: &quot;install&quot;,
            &quot;id&quot;: &quot;5152439&quot;,
            &quot;version&quot;: &quot;::apt&quot;,
            &quot;url&quot;: &quot;https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463&quot;
        }
    ]
}
12:11:52 INFO  Started operation: {'creationTime': '2022-05-20T12:11:52.229Z', 'deviceId': '19392643', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409399', 'id': '19409399', 'status': 'PENDING', 'description': &quot;Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]&quot;, 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
12:11:52 WARN  , retrying in 0.5 seconds...
12:11:52 WARN  , retrying in 0.5 seconds...
12:11:53 WARN  , retrying in 0.5 seconds...
12:11:53 WARN  , retrying in 0.5 seconds...
12:11:54 WARN  , retrying in 0.5 seconds...
12:11:54 WARN  , retrying in 0.5 seconds...
12:11:55 WARN  , retrying in 0.5 seconds...
12:11:55 WARN  , retrying in 0.5 seconds...
12:11:56 WARN  , retrying in 0.5 seconds...
12:11:56 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py&quot;, line 1294, in __call__
    self.testObj.execute()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py&quot;, line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py&quot;, line 232, in fun
    return caller(func, *(extras + args), **kw)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py&quot;, line 74, in retry_decorator
    logger)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py&quot;, line 33, in __retry_internal
    return f()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py&quot;, line 28, in assert_install_in_download_path
    assert &quot;rolldice&quot; in os.listdir(f&quot;{install_directory}&quot;)
AssertionError
12:11:56 WARN  AssertionError: ... blocked 
12:11:56 INFO  
12:11:56 INFO  cleanup:
12:11:57 INFO  Executed sudo, exit status 0
12:11:58 INFO  Executed sudo, exit status 0
12:12:02 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:12:02 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
12:12:02 INFO  Delaying execution by 7 seconds
12:12:23 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
12:12:27 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
12:12:28 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
12:12:28 INFO  
12:12:28 INFO  Test duration: 83.42 secs
12:12:28 INFO  Test final outcome:  BLOCKED
12:12:28 INFO  Test outcome reason: AssertionError:
12:12:28 INFO  
</system-out>
	</testcase>
</testsuite>
